Sources/UtiliKit/ActiveLabel/ActiveLabel.swift

```diff
@@ -0,0 +1,324 @@
+                                            finalLineTrailingInset: 0,
+                                            finalLineLength: 0,
+                                            loadingViewColor: ActiveLabel.loadingGray,
+                                            loadingLineHeight: 8,
+                                            loadingLineVerticalSpacing: 14,
+                                            loadingAnimationDuration: 2.4,
+                                            loadingAnimationDelay: 0.4)
+        }
+    }
+    
+    /// Used to represent an `ActiveLabel`s state.
+    enum State: Equatable {
+        case loading
+        case text(String)
+        
+        static func ==(lhs: State, rhs: State) -> Bool {
+            switch (lhs, rhs) {
+            case (.loading, .loading):
+                return true
+            case (let .text(leftString), let .text(rightString)):
+                return leftString == rightString
+            case (.loading, .text),
+                 (.text, .loading):
+                return false
+            }
+        }
+    }
+    
+    /// The number of activity lines to display. Default is 1.
+    @IBInspectable public var estimatedNumberOfLines: UInt = 1
+    /// Trailing Inset for the last activity line. Default is 0.
+    @IBInspectable public var finalLineTrailingInset: CGFloat = 0
+    /// Line Length in points for the last activity line. If `finalLineTralingInset` is set to greater than 0 this value is not used. Default is 0.
+    @IBInspectable public var finalLineLength: CGFloat = 0
+    /// This color is the darkest area of the line seen during activity animation. Default is (233,231,237) Gray.
+    @IBInspectable public var loadingViewColor: UIColor = ActiveLabel.loadingGray
+    /// The height of each activity line. Default is 8.
+    @IBInspectable public var loadingLineHeight: CGFloat = 8
+    /// Vertical spacing between each activity line when 2 or more lines are displayed. Default is 14.
+    @IBInspectable public var loadingLineVerticalSpacing: CGFloat = 14
+    /// The duration of the gradient animation applied to the activity lines. Default is 2.4.
+    @IBInspectable public var loadingAnimationDuration: Double = 2.4
+    /// The delay that is applied before each animation begins. Default is 0.4.
+    @IBInspectable public var loadingAnimationDelay: Double = 0.4
+    /// Shows the loading views in the Storyboard by default since text of a label can't be nil in a Storybaord. Default is true.
+    @IBInspectable private var showLoadingViewsInStoryboard: Bool = true
+    
+    /// Used to make sure the gradient is centered during snapshot testing
+    var isSnapshotTesting: Bool = false
+    
+    /// Used for Animation KeyPath
+    private static let locationKeyPath = "locations"
+    
+    /// When true the gradient will show centered so that it can be adjusted easily in IB
+    private var isDisplayingInStoryboard: Bool = false
```

Comment:
Removed isSnapshotTesting and isDisplayedInStoryboard var's in favor of a private var for isGradientCentered. It is now set to true in prepareForInterfaceBuilder if the text string in IB is non nil since you can't have a nil text value in IB. Also, created public function configureForSnapshotTest() which will also set the gradient to be centered for snapshots.